components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1.common/src/gen/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/dto/APIDTO.java

```diff
@@ -275,7 +275,8 @@ public static AccessControlEnum fromValue(String v) {
     @XmlEnum(String.class)
     public enum EndpointImplementationTypeEnum {
         INLINE("INLINE"),
-        ENDPOINT("ENDPOINT");
+        ENDPOINT("ENDPOINT"),
+        MOCKED_OAS("MOCKED_OAS");
```

Comment:
@tharindu1st @malinthaprasan This would be the endpoint type when mock implementation is generated from OAS examples. PRAM